ultrasync/main.py

```diff
@@ -572,6 +572,55 @@ def set_alarm(self, areas=None, state=AlarmScene.DISARMED):
 
         return not has_error
 
+    def set_zone_bypass(self, zone, state=False):
+        """
+        Sets zone bypass
+
+        """
+        if not self.session_id and not self.login():
+            return False
+
+        if not isinstance(zone, int) or zone - 1 not in self.zones.keys():
+            logger.error(
+                '{} is not valid zone'.format(zone))
+            return False
+
+        # A boolean for tracking any errors
+        has_error = False
+
+        # Start our payload off with our session identifier
+        payload = {
+            'sess': self.session_id,
+        }
+
+        if self.vendor in (NX595EVendor.ZEROWIRE, NX595EVendor.XGEN,
```

Comment:
Are you certain these three versions support the zone configuration? Usually each have subtle differences. Which model do you have?

Reply:
I have only tested xgen8 (and hoped that the other might work, since they seem similar).

Maybe it's better to be on the safe side, and just allow xgen8, until someone can test it on other models.